tensorbay/client/requests.py

```diff
@@ -12,6 +12,7 @@
 """
```

Comment:
is a `feat` or a `fix`

Reply:
In my opinion, it is a new feature to support multiprocesses security.